lib/writer.js

```diff
@@ -3,16 +3,19 @@
  */
 
 const fs = require('fs')
+const { kill } = require('process')
```

Comment:
`process` is a native global

Reply:
and `kill` isn't used

Reply:
I don't remember putting that there. Whoops